packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json

```diff
@@ -114,6 +114,7 @@
   "ovhManagerSupport_new_creation_tip_forum": "Notre forum :",
   "ovhManagerSupport_new_creation_tip_2": "Vous avez trouvé la réponse à votre demande entre temps ?",
   "ovhManagerSupport_new_creation_tip_3": "Dans ce cas, nous vous invitons à <a ui-sref=\"{{ state }}\">clôturer votre ticket</a> directement depuis votre espace client.",
+  "ovhManagerSupport_new_creation_optional": "{{label}} (facultatif)",
```

Comment:
Why do we need to translate `{{label}}`?

Reply:
We don't, it's a variable 🤔 

It used this to convey context about how `optional` is going to be used

Reply:
@FredericEspiau Thanks for your answer!